### ISSUE TITLE
Unknown 4xx client error codes

### ISSUE THREAD

**Issue by dragosholban** [allinurl/goaccess#1072]
Hi! I installed the latest goaccess code from the master branch but my stats have a lot of "Unknown" "4xx Client Error". I know for sure that most of them are 429 error codes, why are they not listed? I attached a sceenshot in case it helps.

Thanks!

<img width="567" alt="screen shot 2018-04-05 at 13 52 10" src="https://user-images.githubusercontent.com/1357968/38362187-93a2a948-38d8-11e8-84c8-2ef64a24736e.png">

**Comment by allinurl** [comment]
Any updates on this?

**Comment by dragosholban** [comment]
Hi! Thanks for helping. I just tested again and it works... maybe there was something wrong with my installed version.
I'm using Ubuntu 16.04. First time I installed using apt-get, then, when I saw i had an older version, I installed from the source. Right after this, it didn't work, but now, after a few days, it shows the 429 errors too... I guess I messed up something :(

Thanks again and sorry for thinking it was a problem from the code.

As a suggestion, maybe it will be better to have some kind of configuration file where somebody could add as many http codes as needed.
